ensime-startup.el

```diff
@@ -117,6 +117,25 @@ that you have read this message.")
   (let ((ensime-prefer-noninteractive t))
     (ensime--maybe-update-and-start orig-buffer-file-name)))
 
+(defun is-dev-version? (version)
+    "It check VERSION string for few patterns coresponded to dev server version string format."
+    (-contains?
+     (-map (lambda (s) (s-contains? s version))
+           '("-M" "-RC" "SNAPSHOT")
+           )
+     t)
+    )
```

Comment:
this looks weird in lisp. I know it's standard in c / scala but in lisp you wouldn't use newlines. If you're finding you need visual cues to see where parenthesis are ending, there are some good Hacks on the ensime.org page.